app/views/modals/loginDialog.html

```diff
@@ -12,6 +12,10 @@ <h3 ng-if="status401" translate>Authorization Required</h3>
       Please sign in with one of the following services:
     </p>
     <div class="row">
+      <div ng-if="instagramLogin" class="{{relayLogin ? 'col-sm-5 col-sm-offset-1' : 'col-sm-6 col-sm-offset-3' }}">
```

Comment:
This change alone is not sufficient. Do you have the expertise to quickly and easily modify it? Or is it just something I should tinker with until I get it right? @OzzieOrca

Reply:
You have to adjust the size of the other columns. I could play with it locally if you aren't getting anywhere. Bootstrap has a grid size of 12 so everything in the row has to add up to 12.

Reply:
I've updated this commit and this PR is ready for review. Thanks! @OzzieOrca